script.js

```diff
@@ -25,33 +25,37 @@ class AudioController {
         this.stopMusic();
         this.victorySound.play();
     }
-    gameover() {
+    gameOver() {
         this.stopMusic();
-        this.gameoverSound.play();
+        this.gameOverSound.play();
     }
 }
 
 class MixOrMatch {
     constructor(totalTime, cards) {
         this.cardsArray = cards;
+        this.totalTime = totalTime;
         this.timeRemaining = totalTime;
-        this.countdown = null;
-        this.cardClickCount = 0;
-        this.busy = false;
         this.timer = document.getElementById('time-remaining')
         this.ticker = document.getElementById('flips');
-        this.totalClicks = 0;
-        this.cardToCheck = null;
-        this.matchedCards = [];

```

Comment:
I removed all game variable resetting/initialization to the `startGame` method since I changed the clicking of overlays to just call `startGame` instead of create a new `MixOrMatch` object.